docs/dictionary/command/mobileAdCreate.lcdoc

```diff
@@ -22,53 +22,36 @@ The name of the ad to create. This is used to reference the ad
+- 'text'
+- 'full screen'
+- 'banner' - This is the default.
 
 
 topLeft:
-The location in pixel of the top left corner of the ad (coordinates are
-in left,top - x,y order). The default is: 0,0
+The location of the top left corner of the ad (coordinates are
+in left,top - x,y order) in pixels. The default is: 0,0
 
 metaData (enum):
-An array of values that are used to target the ad. The keys are as
-follows: 
-
--   "phone number": The user's mobile number (MSISDN format with
-    international prefix).
--   "keywords": Keywords relevant to the user's specific session. This
-    is comma separated without spaces.
--   "gender": The expected gender of the target user. The allowed values
-    are M, m, Male, F, f, Female.
--   "distribution id": The distribution Channel ID (559 for banner ads
-    and full screen ads, 600 for text ads).
--   "refresh": A value in seconds that defines how often the ad should
-    refresh. Values are between 30 and 300 seconds. The default is 120
-    seconds. 
+An array of values that are used to target the ad. The keys 
+are as follows: 
+
+-   "phone number": The user's mobile number (MSISDN format with international prefix).
+-   "keywords": Keywords relevant to the user's specific session. This is comma separated without spaces.
+-   "gender": The expected gender of the target user. The allowed values are M, m, Male, F, f, Female.
+-   "distribution id": The distribution Channel ID (559 for banner ads and full screen ads, 600 for text ads).
+-   "refresh": A value in seconds that defines how often the ad should refresh. Values are between 30 and 300 seconds. The default is 120 seconds. 
 -   "age": An integer defining the expected age of the target user.
--   "coordinates": GPS ISO code location data in latitude, longitude
-    format. - This is only available on iOS.
--   "location": A comma delimited list of country, state/province, city.
-    - This is only available on iOS.
+-   "coordinates": GPS ISO code location data in latitude, longitude format. - (iOS only)
+-   "location": A comma delimited list of country, state/province, city. - (iOS only)
 
 
 The result:
-If successful, the result is empty.
-The result contains: not registered with service, if you did not
-register your app id using mobileAdRegister before calling
-<mobileAdCreate>. 
-
-The result contains: ad already exists, if an ad of the given name
-already exists.
-
-The result contains: could not create ad, if the ad could not be
-created. This is the case, if the app does not have Internet permissions
-or the registered app key is not valid.
+-   'empty': If successful.
+-   'not registered with service': if you did not register your app id using <mobileAdRegister> before calling <mobileAdCreate>. 
+-   'ad already exists': if an ad of the given name already exists.
+-   'could not create ad': if the ad could not be created. This is the case, if the app does not have Internet permissions or the registered app key is not valid.
 
```

Comment:
I think the formatting here might need more thought, but for now it'll probably do. `empty` should not be enclosed in `'`s, as it is the value empty rather than the string `"empty"`. I would rather you used double-quotes for the other values. Does wrapping not work with lists? I can't remember.